rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -330,6 +343,25 @@ void voteCommitFederation_commit1MemberFederation_returnsInsufficientMembersResp
         assertEquals(FederationChangeResponseCode.INSUFFICIENT_MEMBERS.getCode(), firstVoteResult);
     }
 
+    @Test
+    void voteCommitFederation_withMismatchedHashes_returnsFederationNonExistentResponseCode() {
+        // Arrange
+        voteAndAssertCreateEmptyPendingFederation();
+        int minMembersRequired = 2;
+        voteAndAssertAddFederatorPublicKeysToPendingFederation(minMembersRequired);
+
+        Keccak256 randomHash = Keccak256.ZERO_HASH;
```

Comment:
```suggestion
        Keccak256 wrongHash = Keccak256.ZERO_HASH;
```

I think `wrongHash` or `nonPendingFederationHash` suits better. Wdyt?

Reply:
Still named as `randomHash`